manifests/charts/aeraki/values.yaml

```diff
@@ -0,0 +1,88 @@
+# Default values for aeraki.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: aeraki/aeraki
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: latest
+
+ISTIOD_ADDR: "istiod.istio-system:15010"
+
+ISTIO_CONFIG_STORE_SECRET: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations:
+  sidecar.istio.io/inject: "false"
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  create: false
+  type: ClusterIP
+  port: 80
+
+ingress:
```

Comment:
Do we need an ingress?

Reply:
But I'm not sure if there's a need for this in the future, so I'll save it for now, and if it's not necessary I can delete it